### DISCUSSION TITLE
Way to edit the assignee, label, project ,, etc

### DISCUSSION THREAD

**Discussion by velmuruganc** [Money-Manager-SaaS/android-money-manager-ex#18] [Q&A] [answered]
Hi @tim-hub ,

How can I edit below highlighted fields using my login? Do I need any spl permission? 
<img width="924" alt="image" src="https://user-images.githubusercontent.com/29807533/144376620-6cc95eb6-dab1-48e3-98ed-d8a1bc493675.png">

Thanks,
Velu

**Comment by tim-hub** [comment, marked as answer]
Thanks, @velmuruganc , You are very welcome to make the contribution for the project, I just grant the permission for you, you supposed to receive an invitation.

**Reply by tim-hub** [reply]
Actually, I was working an another project to transfer MMEX db to psql and build a open source SaaS to power it.

It is a web project (React, Node)
